lookbook/docs/patterns/19-attribute-help-texts.md.erb

```diff
@@ -0,0 +1,188 @@
+
+Attribute help texts are a feature that allow administrators to provide
+additional information for attributes of certain entities in OpenProject.
+Currently, work packages and projects are supported.
+
+Our [System admin guide][system_admin_guide] provides guidance for
```

Comment:
This is not a valid link
```suggestion
Our [System admin guide](https://www.openproject.org/docs/system-admin-guide/attribute-help-texts/) provides guidance for
```

Reply:
I'll double-check the syntax, but this should be valid [Reference-Style link](https://www.markdownguide.org/basic-syntax/#reference-style-links), i.e. the URL is in the footnotes.